From 7d641bf5848722babdc7b0e1a9259b7327afffce Mon Sep 17 00:00:00 2001
From: "Jory A. Pratt" <anarchy@gentoo.org>
Date: Mon, 6 Apr 2020 19:16:36 +0200
Subject: [PATCH 02/42] Fortify sources properly

Ensure we fortify properly, features.h is pulled in via fortification.

Signed-off-by: Thomas Deutschmann <whissi@gentoo.org>
---
 config/system-headers.mozbuild                     | 1 -
 media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/config/system-headers.mozbuild b/config/system-headers.mozbuild
index 8942b6244e..a2fd6daf3e 100644
--- a/config/system-headers.mozbuild
+++ b/config/system-headers.mozbuild
@@ -229,7 +229,6 @@ system_headers = [
     'execinfo.h',
     'extras.h',
     'fcntl.h',
-    'features.h',
     'fenv.h',
     'ffi.h',
     'fibdef.h',
diff --git a/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h b/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h
index 6e4dd8c645..bbfa95f138 100644
--- a/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h
+++ b/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h
@@ -20,8 +20,5 @@
 typedef uint8_t    tinybool;
 typedef unsigned short ushort;
 typedef unsigned long  ulong;
-#ifndef __GNUC_STDC_INLINE__
-#define inline
-#endif
 
 #endif /* _SDP_OS_DEFS_H_ */
-- 
2.31.1

